numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -928,25 +928,24 @@ def add_newdoc(place, name, doc):
 
     The Heaviside step function is defined as::
 
-                             0   if x < 0
-        heaviside(x, h0) =  h0   if x == 0
-                             1   if x > 0
```

Comment:
I prefer the original also, but that can wait until the generation of the signature is improved so we can make it agree with the rest of the docstring.